debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsSnapshotProducer.java

```diff
@@ -148,7 +151,7 @@ private void takeSnapshot(Consumer<SourceRecord> consumer) {
             // we're locking in SHARE UPDATE EXCLUSIVE MODE to avoid concurrent schema changes while we're taking the snapshot
             // this does not prevent writes to the table, but prevents changes to the table's schema....
             schema.tables().forEach(tableId -> statements.append("LOCK TABLE ")
-                                                         .append(tableId.toString())
+                                                         .append(tableId.toQuotedId(TableIdTransformer.DOUBLE_QUOTED))
```

Comment:
Sorry @gunnarmorling, I had just commented on this on DBZ-298 - https://github.com/debezium/debezium/pull/257#discussion_r129597318 - I agree, its unfortunate that the two commits got intermingled.  I did DBZ-298 and then branched DBZ-297 (which made sense at the time, I checked on Gitter before doing it).